### PR TITLE
Custom fields act as search filters by default

### PR DESCRIPTION
**Pull request by ovan**
The default used to be false but this PR changes the default option to be true.

### CODE REVIEW

File: app/views/admin/custom_fields/form/_search_filter.haml [hunk 1]
```diff
@@ -4,13 +4,13 @@
       - custom_field.with_type do |type|
         - case type
         - when :date_field
-          = form.check_box :search_filter, disabled: true
+          = form.check_box :search_filter, checked: false, disabled: true
```
Comment (rap1ds) [inline app/views/admin/custom_fields/form/_search_filter.haml]: Why is this needed?

Comment (ovan) [inline app/views/admin/custom_fields/form/_search_filter.haml]: It just looks more logical that when text or date fields cannot be made a filter and the box is shown as disable the checkbox is not checked.
